core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -890,6 +981,49 @@ class DAGScheduler(
     submitStage(finalStage)
   }
 
+  // This method is used to handle jobs that contain a closure for setting configurations on driver
+  private[scheduler] def handleJobSubmitted(jobId: Int,
```

Comment:
same thing here -- don't need this version, just change the existing one to always expect the extra arg